### ISSUE TITLE
[EN][RAID][Firelands][Baleroc]Baleroc encounter bugs

### ISSUE THREAD

**Issue by Aival94** [Atlantiss/BugTracker#8894]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: I will describe all Baleroc encounters bugs that i know.

**Current behaviour**: 
1) Sometimes the Crystals on [Baleroc](http://www.wowhead.com/npc=53494/baleroc) fall down(does not do the aoe damage but still the healers cannot get any stack as the crystal does not link to anyone as well).
2) Sometimes the Crystal does not target the closest target and continue giving stacks and dealing damage to the previous one who was there. (This might happen because of some classes cooldowns(we had Rogue going to get stacks while a druid was moving away))
3) Seems that [Baleroc](http://www.wowhead.com/npc=53494/baleroc) during its blade abilities([Decimation Blade](http://www.wowhead.com/spell=99352/decimation-blade) and [Inferno Blade](http://www.wowhead.com/spell=99350/inferno-blade)) is also auto-attacking.(or sometimes the [Blaze of Glory](http://www.wowhead.com/spell=99252/blaze-of-glory) does not increase the Current health but only the Maximum health).
4) If one of [Countdown](http://www.wowhead.com/spell=99516/countdown) target die it cannot be removed or not explode immediatelly but wait for the countdown to of the other target.
5) [Torment](http://www.wowhead.com/spell=99256/torment) seems to be scaling with the boss damage as while the encounter continue the damage seems to increase.
6) Sometimes after [Vital Flame](http://www.wowhead.com/spell=99263/vital-flame) fade away there is a small possibility to lose all the stacks of [Vital Spark](http://www.wowhead.com/spell=99262/vital-spark)(Really not sure how to reproduce it).
7) (Happened only once) The [Torment](http://www.wowhead.com/spell=99256/torment) was dealing damage like the player had Debuff [Tormented](http://www.wowhead.com/spell=99257/tormented) but did not have it.
8) (For Test) Seems that after the last fix [Anti-Magic Shell](http://www.wowhead.com/spell=48707/anti-magic-shell) can still decrease the damage of [Decimation Blade](http://www.wowhead.com/spell=99352/decimation-blade) (will try to test it as soon as i can).

**Expected behaviour**: What should happen instead
1) The crystals should not fall down.
2) The crystals should always target the closest target.
3) Should not use auto-attacks while blades active (or [Blaze of Glory](http://www.wowhead.com/spell=99252/blaze-of-glory) should increase Current Health by the same amount it increase the Maximum Health).
4) [Countdown](http://www.wowhead.com/spell=99516/countdown) should probably just explode instantly(so is just minor) or need to stay on dead target until it get removed somehow.
5) [Torment](http://www.wowhead.com/spell=99256/torment) should not scale with the boss damage.
6) [Vital Spark](http://www.wowhead.com/spell=99262/vital-spark) should never be lost until you die or finish Baleroc encounter.
7) (Happened only once) [Torment](http://www.wowhead.com/spell=99256/torment) should deal normal Damage while the target do not have Debuff [Tormented](http://www.wowhead.com/spell=99257/tormented)
8) (For Test) [Anti-Magic Shell](http://www.wowhead.com/spell=48707/anti-magic-shell) should not decrease the damage of [Decimation Blade](http://www.wowhead.com/spell=99352/decimation-blade).

**Steps to reproduce the problem**:

1.  Enter Firelands.
2.  Engage in battle [Baleroc](http://www.wowhead.com/npc=53494/baleroc) many times
3.  See the bugs.

**Revision**: 5304

**Comment by Illusie** [comment]
You right.
Shard sometimes fall through the texture, healers unable to get more Vital Spark.
Sometimes Shard ignoring a player (even if he stand close to it).

Auto attacks still happen before Decimation Blade
And also Inferno Blade.
If i red well when he cast those abilities Baleroc stop auto-attacking.

Countdown, you right if the linked-player is dead it explode anyway.
Didn't noticed loss of Vital Spark.
Didnt ever got damage from Tormented

In a Point of View of BDK Tank:

I don't think AMS reduce the damage of Decimation Blade.
Atleast when I play my Tank. I use AMS against Inferno Blades and not against Decimation Blade.

Since Decimation Blade cannot be dodged & parried. I even tried one time to use Rune Dancing to see if it dodge or parry it didn't, atleast not to me.
Icebound Fortitute I dont see a real usage for it in Baleroc fight.

Thanks for the reports, keep doing amazing job.

**Comment by Vayns** [comment]
Extra info for point 2. Can confirm this for rogues. If you use vanish after the crystal has spawned and try to take it it will ignore you. When a new crystal spawns it will no longer ignore the rogue though.

@Illusie you can def. parry/dodge decimation blade, have seen tanks nog getting hit by 1 decimation for an entire fight.

**Comment by Aival94** [comment]
@Illusie i know that normally it should not decrease and i usually do not use AMS for Decimation but yesterday missclicked it and i saw my Health be on 75% after a hit(maybe just the timing of my healers was so good that they healed me after the decimation blade to this amount but i think with their stacks and ilv they should not be able to do it so fast but did not manage to find this specific logs so i cannot be sure so need testing).

**Comment by Prodiege** [comment]
@ #2 and @ Vayns. If a rogue vanishes even when the animation to spawn the crystal is out (even though the crystal is not yet out) you also won't get targeted

**Comment by Kozakko** [comment]
Also Countdown interupts casts when it appears.. (casts of linked players to be precise)

**Comment by Aival94** [comment]
Implemented [Countdown](http://www.wowhead.com/spell=99516/countdown) interruption by @Kozakko.

Unfortunatelly for the AMS my dk has id and all the dks on my guild have as well so the update will happen when i will tank Baleroc again on my dk or one of my guildies will try it.(Or when Jenova test it).

**Comment by Illusie** [comment]
first im sorry for talking about decimation blade here :D.
since decimation blade is most interesting me in Baleroc fight.
so actually this message is off-topic i have nothing new to say.

@vayns vayns, when im reading the info of Decimation blade I must admit it confuse me,

"**Melee** strikes deal **Shadow** damage equal to 90% of the target's maximum health, but always at least XXXXX damage. This damage cannot be **resisted** or mitigated by **normal means**."

It first open that Decimation Blade is "Melee strike" it means (maybe?) you actually can dodge or parry it because it "Melee strike" but then it says "the damage cannot be resisted or mitigated by normal means"

This part I understand that the damage cannot be 'resisted' by my interpretation i read it that no matter what you will 100% get hitted by it.

Than right after it says 'by normal means' - clearly I have no clue what they meant to say.

And the most funny part of the description of the Ability is the part that says 'Shadow damage'
IT means (maybe?) AMS support to lower the damage taken? but wait isn't that attack suppost to hit you 90% of your health no matter what?

Someone can help me because I'm lost?

**Comment by Aival94** [comment]
@Illusie Please check this one: "This means that the Decimation tank can take a total of 3 strikes that do >240K damage. This damage cannot be reduced by cooldowns, absorbs, or blocks, but fortunately it can be dodged or parried." - *Source*: [Baleroc](http://wow.gamepedia.com/Baleroc)

**Comment by Illusie** [comment]
@Aival94 Ok, I never seen this one before, I'll check if I can dodge or parry it next ID.

**Comment by Psychobird7** [comment]
To add to this. 

Inferno Blade should be blockable (by paladins/warriors)
. 
http://www.wowhead.com/npc=53494/baleroc#comments:id=1471272 
"Inferno Blade can be dodged, parried and blocked."   

Video from retail: https://www.youtube.com/watch?v=Pll_qZ_gwQw 
Check the combat text on the left side of the screen.   

Log from retail: http://worldoflogs.com/reports/p6uh36yxpv1xxncu/details/3/?s=4018&e=4198 
Go into "Damage by Spell" --> Taken --> Inferno Blade. "blocked".

**Comment by Aival94** [comment]
Updated with AMS and Inferno Blade reported by @Phychobird7 here https://github.com/Atlantiss/BugTracker/issues/8914 and linked by @Piklo.

**Comment by Jenovas** [comment]
Can someone give me some proof that AMS shouldnt absorb the Decimation DMG?
For me AMS works just the same way PWS works, it's an absorb, not a resist. The spell description talks about not being able to resist it doesnt say anything about not being able to absorb.

I just checked Blizzard Forums and there they claim that Decimation Blade can be absorbed.

**Comment by Aival94** [comment]
@Jenovas you are right and sorry for the wrong report. I though AMS is working like a resist to magic attacks. I was searching as well just now to find this post as well: http://www.mmo-champion.com/threads/971226-When-to-use-AMS-DRW-on-Bale-roc?p=13074031&viewfull=1#post13074031
"So many people are confused (as I was when I first tanked it).
Avoidance - When you completely avoid an attack. IE Dodge, Parry and Miss.
Mitigation - When the all damage is reduced by a flat percent. Normal CDs like IBF and Bone Shield.
Resistance - When damage is reduced for a specific type of damage. Spells and Physical.
Absorption - When damage is absorbed. IE AMS, PW:S and Pali Mastery."

So yeah AMS should be able to absorb a % of the Decimation Blade(the 50% of the DK Max Health) so is working just fine. Furhthermore the [Power Word: Shield](http://www.wowhead.com/spell=17/power-word-shield) and Paladins Mastery [Illuminated Healing](http://wow.gamepedia.com/index.php?title=Mastery:_Illuminated_Healing&oldid=2818649) should work as well. 
Sorry again for the mistake. (already removed AMS as bugged).

**Comment by Aival94** [comment]
Implemented as #10 the spots where Baleroc can fall down after is dead.

**Comment by stanq87** [comment]
thres this bug www.youtube.com/watch?v=OkNOCZ708s0 when after wipe he's in combat on someone

**Comment by Aival94** [comment]
Updated as #11 the Baleroc hitting the dead body.(have not seen it for sometime so i thought it was fixed xD)

**Comment by qwertyasad** [comment]
Baleroc deals damage during Decimating Strike, but it doesn't show anywhere in the combat log. Your health just falls down for no reason.

You can see in the screen that during Baleroc charging Decimating Strike, I'm overhealed (meaning that I have 100% health), then after a moment I take the hit and die. That's because my health has fallen below 90% between the heal and the hit, but as you can see, it doesn't show in the log.

P.S. Loom of Fate is Spidersilk Spindle's trigger, and it doesn't cause the bug, as later I switched it and same thing happened.

![capture](https://cloud.githubusercontent.com/assets/26233433/23705506/7dd28ace-040a-11e7-86af-115faa94dff0.PNG)

**Comment by stanq87** [comment]
more about what qwertyasad wrote
http://i.imgur.com/0hhqC6n.jpg

**Comment by Prodiege** [comment]
@qwertyasad 
@stanq87 

I have had this happen a lot of times. I have a strong suspicion it is either something to do with the bonus health from Ancestral Healing talent falling off from shamans or being overwritten from a priest that has Inspiration

**Comment by stanq87** [comment]
well efects like that should when they  run out  "cut out" the top HP pull, so if u have a base hp of 100k, and u get 40k from shaman  so if your hp is 120k when the efect is lost u have 100k not 80k - or something like that

**Comment by Aival94** [comment]
Not really @stanq87 . The HP you get is % of your max health and should decrease again to the same %. Furthermore for the shaman i have done some tests and while in combat the health return where it used to be(for example was 80% and it return on 80%). The only othe thing that make be the case might be that these kind of spells together with baleroc one cause and difference between the after the activation of spell/buff and after the fading of the spell/buff health.

**Comment by stanq87** [comment]
yes, BUT the shaman hp increase is applied when shaman over heals someone, so the mechanic  should be more like the pvp trinkets that give u extra hp,   when it run outs u loose the extra hp not base hp, SO lest say u have 100k hp, the buff gets u +10% so u have buffed 110k u get hit 15 k , u have 95k ( u have 86% hp) so if then u loose the buff u shoul loose the extra hp amount and have 95k not 86k,  BESIDES this thing on baleroc happens even when u dont have a shaman healer in the group to all tanks, but most commonly to DK's

**Comment by Aival94** [comment]
It happen mostly to DKs because almost all groups use DKs for Baleroc as is much much easier to heal a DK because of its self heal and a dk can be the easiest hybrid tank to deal 20k damage. 
About the ability should remove again % so if you have 100k and get a 10% buff you go to 110k but if you lose 22k(20%) and be at 88k it should move you to 80k(80%) when you lose the buff but i need to check if i am correct about that or was later implemented to work like that.
Furthermore this can happen even if you have paladins only so i think is probably a bug with autoattack during the blades abilities.

**Comment by Aival94** [comment]
Removed Countdown interruption(fixed on Rev. 5344) and updated rev.

**Comment by Aival94** [comment]
Implemented stacks of Vital Flame only for 25 HC and updated rev.

**Comment by stanq87** [comment]
PT3, i had something difrent on baleroc: my tank had 450k hp 100%, and he had that up until he got a shot from deciamte but just before deciamte something doubled his hp pool to 900k but hes % stayed on 450k so at 50% aaannnnd then he died from deciamte [log](https://www.youtube.com/watch?v=P8sHt0HsLCA) , this happend to a feral druid with a holy paladin and resto druid  , and its not  blaze of glory , those can give 20% more hp not 100%

**Comment by Aival94** [comment]
Probably Frenzied Regeneration with some rallying cry?

**Comment by stanq87** [comment]
a 822k decimate on a 810k hp tank , so tank had 99% hp and died shiiiieeeeet - [img](https://i.imgur.com/AlhDKvn.jpg)

**Comment by stanq87** [comment]
doesent bareloc as a dual wielding boss should have a smaller hit cahnce than other ? so not 4,5% but more like 20% ?

**Comment by stanq87** [comment]
#9912
